src/cascadia/TerminalSettingsModel/ActionArgs.cpp

```diff
@@ -709,23 +709,40 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
 
     winrt::hstring SuggestionsArgs::GenerateName() const
     {
-        auto base{ RS_(L"SuggestionsCommandKey") };
-        switch (Source())
+        std::wstringstream ss;
+        ss << RS_(L"SuggestionsCommandKey").c_str();

```

Comment:
nit: `std::wstring_view{ RS_(L"SuggestionsCommandKey") }`.
You can also just use `std::wstring::append()` instead of using `wstringsteam`, because string streams are fairly hefty.